config/sync/user.role.site_manager.yml

```diff
@@ -7,23 +7,74 @@ label: 'Site Manager'
 weight: -3
 is_admin: null
 permissions:
+  - 'access administration pages'
   - 'access content overview'
+  - 'access contextual links'
+  - 'access environment indicator'
   - 'access file_browser entity browser pages'
+  - 'access files overview'
   - 'access image_browser entity browser pages'
+  - 'access media overview'
   - 'access media_browser entity browser pages'
   - 'access patterns page'
+  - 'access site in maintenance mode'
+  - 'access taxonomy overview'
+  - 'access user profiles'
+  - 'access video_browser entity browser pages'
+  - 'administer footer menu items'
   - 'administer main menu items'
   - 'administer media'
```

Comment:
- This gives access to `/admin/config/media/media-settings` path. i dont think we'd want them in there. we might have to change the permission on that specific path